### ISSUE TITLE
Arrange accessibility retest - 4 advisories

### ISSUE THREAD

**Issue by sarawilcox** [nhsuk/nhsuk-service-manual#2637]
There were the 4 advisories for us to consider: https://alpha.hugr.app/view/becd7a

- [DIG011: Text alternative for image should be improved.](https://alpha.hugr.app/view/becd7a#DIG011) Fixed https://service-manual.nhs.uk/design-system/patterns/find-bsl-content
- [DIG234: Table header cell needs scope attribute](https://alpha.hugr.app/view/becd7a#DIG234) Fixed https://service-manual.nhs.uk/design-system/components/table#how-it-works
- [DIG215: Anchors are used for buttons](https://alpha.hugr.app/view/becd7a#DIG215) - decided not to action
- [DIG308: Instructions for completing the form are marked up inappropriately](https://alpha.hugr.app/view/becd7a#DIG308) - still to fix (see #2497 )

Arrange a retest with Dig when we've finished updating file upload.
